tests/test_shell_zprofile_synthetic_login.py

```
@@ -0,0 +1,113 @@
+    env["HOME"] = str(home)
+    env["ZDOTDIR"] = str(wrapper_dir)
+    env["CMUX_ZSH_ZDOTDIR"] = str(orig)
+    env["CMUX_ZSH_SOURCE_LOGIN_PROFILE"] = "1"
+    env["CMUX_SHELL_INTEGRATION"] = "0"
+
+    args = ["zsh", "-d", "-i"]
+    if login:
+        args.append("-l")
+    args.extend(["-c", "true"])
+
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)